Description: test_backup_s3.py was added late in the cycle
  and there is not currently a package in Ubuntu for
  python3-moto. Skip the tests for now.
Author: Corey Bryant
Forwarded: no

--- a/cinder/tests/unit/backup/drivers/test_backup_s3.py
+++ b/cinder/tests/unit/backup/drivers/test_backup_s3.py
@@ -23,10 +23,14 @@
 import tempfile
 import threading
 from unittest import mock
+import unittest
 import zlib
 
 from eventlet import tpool
-from moto import mock_s3
+try:
+    from moto import mock_s3
+except ModuleNotFoundError:
+    mock_s3 = unittest.skip("moto library is not available")
 from oslo_utils import units
 
 from cinder.backup.drivers import s3 as s3_dr
